includes/batches/class-batch-comments.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Comments batch class.
+ *
+ * @package Locomotive/Batch
+ */
+
+namespace Rkv\Locomotive\Batches;
+
+use WP_Comment_Query;
+use Rkv\Locomotive\Abstracts\Batch;
+
+/**
+ * Batch Comments class.
+ */
+class Comments extends Batch {
+	/**
+	 * The individual batch's parameter for specifying the amount of results to return.
+	 *
+	 * @var string
+	 */
+	public $per_batch_param = 'number';
+
+	/**
+	 * Default args for the query.
+	 *
+	 * @var array
+	 */
+	public $default_args = array(
+		'number' => 10,
+		'offset' => 0,
+		'no_found_rows' => false,
```

Comment:
Is `no_found_rows` responsible for determining the total count? If so, would we need to keep this? (and maybe on other queries too?)